python/acquire/acquire.pyi

```diff
@@ -56,35 +58,39 @@ class CameraProperties:
     shape: Tuple[int, int]
     input_triggers: InputTriggers
     output_triggers: OutputTriggers
+
     def __init__(self, *args: None, **kwargs: Any) -> None: ...
     def dict(self) -> Dict[str, Any]: ...
 
 @final
 class Capabilities:
     video: Tuple[VideoStreamCapabilities, VideoStreamCapabilities]
+
     def __init__(self, *args: None, **kwargs: Any) -> None: ...
     def dict(self) -> Dict[str, Any]: ...
 
 @final
-class ChunkDims:
-    width: int
-    height: int
-    planes: int
-    def dict(self) -> Dict[str, Any]: ...
+class DimensionType:
+    Space: ClassVar[DimensionType]
+    Channel: ClassVar[DimensionType]
+    Time: ClassVar[DimensionType]
+    Other: ClassVar[DimensionType]
```

Comment:
Do we have documentation that might guide users on which one to pick?

In my head, these affect ome-zarr metadata and (in the future maybe)  impact what we do with on-the-fly lod's. e.g. We might downsample in space and time but not channel.

Reply:
All we have is this in the acquire-driver-zarr readme:

> The first two dimensions should represent the width and height of the frame, respectively. The array_size_px for these dimensions should match the width and height of the frame, and the kind field should be DimensionType_Space. The rest of the dimensions should match the order of acquisition.

> We might downsample in space and time but not channel.

Yeah exactly. I can add docstrings in acquire.pyi